clang-tools-extra/clangd/refactor/Rename.cpp

```diff
@@ -681,12 +957,22 @@ renameOutsideFile(const NamedDecl &RenameDecl, llvm::StringRef MainFilePath,
            ExpBuffer.getError().message());
       continue;
     }
+    std::string RenameIdentifier = RenameDecl.getNameAsString();
```

Comment:
Technically `RenameIdentifier` isn’t correct here because this can be an Objective-C selector name, right? Which would be multiple identifiers and colons.